mmv1/templates/terraform/custom_expand/containerattached_cluster_authorization_user.go.erb

```diff
@@ -17,26 +17,38 @@ type attachedClusterUser struct {
     Username string     `json:"username"`
 }
 
+type attachedClusterGroup struct {
+		Group string				`json:"group"`
+}
+
 // The custom expander transforms input into something like this:
 // authorization {
 //    admin_users [
 //      { username = "user1" },
 //      { username = "user2" }
 //    ]
+//    admin_groups [
+//      { group = "group1" },
+//      { group = "group2" },
+//    ]
 // }
 // The custom flattener transforms input back into something like this:
 // authorization {
 //    admin_users = [
 //      "user1",
 //      "user2"
 //    ]
+//    admin_groups = [
+//      "group1",
+//      "group2"
+//    ],
 // }
 func expand<%= prefix -%><%= titlelize_property(property) -%>(v interface{}, d tpgresource.TerraformResourceData, config *transport_tpg.Config) (interface{}, error) {
-  	l := v.([]interface{})
+  l := v.([]interface{})
```

Comment:
```suggestion
	l := v.([]interface{})
```

nit: mix of tabs and spaces in whitespace

Reply:
I'll go ahead and commit this and then merge the PR after

Reply:
Thank you!